src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_TupleBinaryOperator.cs

```diff
@@ -184,7 +184,7 @@ private bool IsLikeTupleExpression(BoundExpression expr, out BoundTupleExpressio
             {
                 case { ConstantValue: { } }:
                     return VisitExpression(expr);
-                case BoundConversion { Conversion: { Kind: ConversionKind.DefaultOrNullLiteral } conversion } bc:
+                case BoundConversion { Conversion: { Kind: ConversionKind.DefaultLiteral } conversion } bc:
```

Comment:
>DefaultLiteral [](start = 74, length = 14)

Should NullLiteral be handled as well?

Reply:
I don't think so. `null` converted to a nullable value type (ie. by a null literal conversion) is handled by the `ConstantValue` case above.

---
In reply to: [314121528](https://github.com/dotnet/roslyn/pull/37596#discussion_r314121528) [](ancestors = 314121528)

Reply:
> I don't think so. null converted to a nullable value type (ie. by a null literal conversion) is handled by the ConstantValue case above.

What about null converted to a reference type?

---
In reply to: [315451052](https://github.com/dotnet/roslyn/pull/37596#discussion_r315451052) [](ancestors = 315451052,314121528)

Reply:
That would also have a constant value.

---
In reply to: [315923328](https://github.com/dotnet/roslyn/pull/37596#discussion_r315923328) [](ancestors = 315923328,315451052,314121528)